packages/ra-data-graphql/src/constants.ts

```diff
@@ -18,4 +18,4 @@ export const MUTATION_TYPES = [
     UPDATE_MANY,
     DELETE_MANY,
 ];
-export const ALL_TYPES = QUERY_TYPES.concat(MUTATION_TYPES);
+export const ALL_TYPES = [...QUERY_TYPES, ...MUTATION_TYPES] as string[];
```

Comment:
I don't understand why you need the `as` here.

Reply:
I don't remember why too. I will remove it